packages/flutter/lib/src/material/app.dart

```diff
@@ -132,9 +133,29 @@ class MaterialApp extends StatefulWidget {
        ),
        super(key: key);
 
-  /// A one-line description of this app for use in the window manager.
+  /// A one-line description used by the device to identify the app for the user.
+  ///
+  /// The title is used by the device to identify the app for the user. On
```

Comment:
you can remove the first sentence here since it almost exactly duplicates the previous one

Reply:
Oops, yes.